cloudflare/src/request.rs

```diff
@@ -0,0 +1,87 @@
+use crate::{Credentials, Endpoint, CLIENT_API_V4_URI};
```

Comment:
I'm actually not sure about having this as a separate file. The original idea was to reduce dependencies by making it optional (and removing the url dependency with it) but it doesn't really seem that worth it now. It's only 2 structs an an extension trait.

Reply:
Agree, it doesn't seem worth its own feature. If this let you avoid pulling in another dependency from crates.io, I'd say, sure, go ahead, hide it behind a feature. But if not, let's not worry?